### PR TITLE
Introduces kpi design update for orders listing page - also includes layout changes for all kpi's in migrated pages

### LINKED ISSUE #15230: [ORDER LIST] New KPI header design
**USER STORY**

As a merchant, I want to quickly check the important KPI of my orders.

**WHY**

The KPI icon and display are redesigned. They were talking to much space and were too separated from each other, preventing a quick reading.

**DESIGN**

https://invis.io/YKTGGFZAMCB#/376656277_Order_List_-_Quick_View

image

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Design changes mostly impacted in new order listing page located in **admin-dev/index.php/sell/orders/orders**. See below for more information of what has been changed :arrow_double_down: 
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #15230
| How to test?  | See below for what has been changed. **Don't forget to rebuilt assets**

new orders page design for kpis:

Screenshot (23)

I mentioned problem here in comments why gaps are still there: https://github.com/PrestaShop/PrestaShop/issues/15230#issuecomment-541032213

other pages were impacted only with layout ( now kpi's are more condensed):

customers page:
Screenshot (24)

categories page:

Screenshot (25)

translations page:

Screenshot (26)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16033)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
Nice job !

File: admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl [hunk 1]
```diff
@@ -39,8 +39,10 @@
       </div>
     {/if}
     <span class="title">{$title|escape}</span>
-    <span cLass="subtitle">{$subtitle|escape}</span>
-    <span class="value">{$value|escape|replace:'&amp;':'&'}</span>
+    <div class="kpi-description">
+      <div cLass="subtitle">{$subtitle|escape}</div>
```
Comment (sarjon) [inline admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl]:
```suggestion
      <div class="subtitle">{$subtitle|escape}</div>
```

Comment (matks) [reply admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl]: Nice catch !

Comment (matks) [reply admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl]: I cant merge the suggestion 😢 github tells me I have no permission to do this

Comment (tomas862) [reply admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl]: I think it was in legacy - super duper hard to see such things ;D

**Review by sarjon — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by TristanLDD** [comment]
Nice, @tomas862 could you fix these spacings: 
Blue

Now we would have to replace all the icons from other impacted pages with material blue icons, but I guess this has to be done in another issue as it doesn't concern order page?

**Comment by tomas862** [comment]
@TristanLDD  yes, done it  :) Here's how it looks like now 
Screenshot (29)

and you are right, we should put it material icons replacement for other pages in another PR :)

**Review by matks — APPROVED** [review]

**Pull request merged**